libs/vkd3d/swapchain.c

```diff
@@ -122,7 +122,7 @@ struct dxgi_vk_swap_chain
          * We don't care about wrap around.
          * We just care about equality check so we can atomically check if all outstanding present events have completed on CPU timeline.
          * This is used to implement occlusion check. */
-        uint32_t present_count;
+        uint64_t present_count;
```

Comment:
Probably need VKD3D_ALIGN(8) for 32-bit. Also, the comment above is now misleading w.r.t. not caring about wraparound.

Reply:
I changed this to `UINT64` since that is naturally aligned.